public/js/themes.js

```diff
@@ -0,0 +1,49 @@
+// Managing theme changes:
+const darkId = 'dark-theme-css'
+const lightId = 'light-theme-css'
+
+function setDarkCss() {
```

Comment:
I really like the dark theme.
Maybe `setDarkCss` and `setLightCss` could be done in one function with a paramteter as they are doing basicly the same thing.

Reply:
Yep, totally true, forgot to do that, hotfix on it's way.